agir/front/components/authentication/Connexion/LoginFacebook.js

```diff
@@ -16,7 +17,7 @@ const LoginFacebook = (props) => {
   return (
     <ButtonFacebook {...props} link color="choose" href={routes.facebookLogin}>
       <img src={facebookImg} width="24" height="24" />
-      &nbsp; Connexion avec Facebook
+      {_("  Connexion avec Facebook")}
```

Comment:
In this case, I think it is better to keep the initial space outside of the function, to avoid not including it in the translated text
```suggestion
      &nbsp;{_("Connexion avec Facebook")}
```